vpr/src/base/vpr_context.h

```diff
@@ -95,6 +95,10 @@ struct TimingContext : public Context {
     std::shared_ptr<tatum::TimingConstraints> constraints;
 
     t_timing_analysis_profile_info stats;
+
+    /* Represents whether or not VPR should fail if timing constraints aren't met. */
+    bool fail_status_if_fail_timing = false;
+    std::string fail_timing_msg = "";
```

Comment:
Does this message have to be in the context?  Ideally contexts are major items read in many places so it would be better to keep this message out of the context if possible.